pom.xml

```diff
@@ -20,6 +24,16 @@
             <artifactId>junit</artifactId>
             <version>4.12</version>
         </dependency>
```

Comment:
You don't have testing in this project, this dependency can be deleted

Reply:
done ✔️